out/production/GeneralaGame/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
Read about .gitignore file. We definitely need one here. These kind of files are not supposed to be part of the source.
Jar, .MF, .class, I will add one for this repo in master branch. Keep in mind that this file is used for ignoring all files that are not needed.